tests/integration/client/sdk/v1/test_datasets.py

```diff
@@ -387,6 +392,32 @@ async def test_get_records(role: UserRole) -> None:
     assert FeedbackItemModel(**response.parsed.items[0].dict())
 
 
+@pytest.mark.skip(reason="unfinished")
```

Comment:
I set this as skipped because it was unfinished, we should remove it.

Reply:
Removing this and create proper sdk unit tests